unit_tests/constants.py

```diff
@@ -0,0 +1,25 @@
+test_simulation = {
+    "username": "user@test.com",
+    "name": "AggregatorTestSuite",
+    "uuid": "some_random_uuid",
+    "domain_name": "https://cool.webpage.com",
+    "websockets_domain_name": "wss://cool.webpage.com/external-ws",
+}
+
+test_agg = {
+    "name": "TestAggr",
+    "uuid": "some_random_aggregator_uuid"
+}
+
+test_batch_command_dict = {"some_device_uuid": [{"command_dict0"},
```

Comment:
At the moment all these constants are used in just one module (`test_aggregator.py`) and they are very specific to it. Why don't we just move them in `test_aggregator.py` directly? If/when they will be needed by other test modules as well, we can move them into a separate `constants.py` module.

What do you think?

Reply:
I agree that the module looks cleaner without those constants. But I would only move them in a separate module if/when they will need to be shared across more than one test module.